.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -424,16 +432,16 @@ jobs:
             --conn-disrupt-dispatch-interval 0ms
 
 
-      - name: Upgrade Cilium in cluster1 and enable kvstoremesh
+      - name: Upgrade Cilium in cluster1
         env:
           KVSTORE_ID: 1
         run: |
           cilium --context ${{ env.contextName1 }} upgrade --reset-values \
             ${{ steps.newest-vars.outputs.cilium_install_defaults }} \
             ${{ steps.vars.outputs.cilium_install_defaults }} \
+            ${{ steps.vars.outputs.enable_clustermesh_failover }} \
             ${{ steps.kvstore.outputs.cilium_install_kvstore }} \
-            ${{ steps.clustermesh-vars.outputs.cilium_install_cluster1 }} \
-            --set clustermesh.apiserver.kvstoremesh.enabled=true
+            ${{ steps.clustermesh-vars.outputs.cilium_install_cluster1 }}
 
       - name: Rollout Cilium agents in cluster2
```

Comment:
Could you delete all occurrences of this step please, as no longer required with the new reconnection mechanism.

Reply:
banished